src/products/utils/data.ts

```diff
@@ -234,3 +317,27 @@ export function mapFormsetStockToStockInput(
     warehouse: stock.id
   };
 }
+
+export const mergeFileUploadErrors = (
+  uploadFilesResult: Array<MutationFetchResult<FileUpload>>
+): UploadErrorFragment[] =>
+  uploadFilesResult.reduce((errors, uploadFileResult) => {
+    const uploadErrors = uploadFileResult.data.fileUpload.uploadErrors;
+    if (uploadErrors) {
```

Comment:
I think if we don't have a defalult `return errors` here, in next iteration errors will come as undefined and spread will break?

Reply:
fixed